src/System.Private.CoreLib/src/System/AppContextConfigHelper.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+
+namespace System
+{
+    internal static class AppContextConfigHelper
+    {
+        internal static int GetConfig(string configName, int defaultValue)
```

Comment:
Overloading just based on the default value's type seems precarious, suggestion: GetIntConfigValue, GetShortConfigValue, etc.